vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -12,124 +12,107 @@
 //
 
 <template>
-  <div class="meta-container__inner">
+  <div class="base-address">
     <!-- Display fields -->
     <v-expand-transition>
-      <div class="address-block"
-          v-if="!editing"
-      >
-        <div class="address-block__info">
-          <div class="address-block__info-row">
-            {{ addressLocal.streetAddress }}
-          </div>
-          <div class="address-block__info-row pre-wrap"
-            v-html="addressLocal.streetAddressAdditional">
-          </div>
+      <div v-if="!editing" class="address-block">
+        <div class="address-block__info pre-wrap">
+          <div class="address-block__info-row">{{ addressLocal.streetAddress }}</div>
+          <div class="address-block__info-row">{{ addressLocal.streetAddressAdditional }}</div>
           <div class="address-block__info-row">
             <span>{{ addressLocal.addressCity }}</span>
-            <span v-if="addressLocal.addressRegion !== '--'">&nbsp;{{ addressLocal.addressRegion }}</span>
-            <span v-if="addressLocal.postalCode !== 'N/A'">&nbsp;&nbsp;{{ addressLocal.postalCode }}</span>
-          </div>
-          <div class="address-block__info-row">
-            {{ getCountryName(addressCountry) }}
-          </div>
-          <div class="address-block__info-row"
-               v-if="addressLocal.deliveryInstructions"
-          >
-            {{ addressLocal.deliveryInstructions }}
+            <span v-if="addressLocal.addressRegion">&nbsp;{{ addressLocal.addressRegion }}</span>
```

Comment:
may be just add a comment why nbsp is used?  is it for spacing needs or some other purpose?

Reply:
This is a non-breaking space and is the proper and usual way to provide real white-space in text in HTML. Often developers "assume" that there will be a space provided when elements have a space character between them or are on different lines ... this is not always true, depending on the type of element and the white-space CSS property. In this case I was explicit about providing a whitespace character in front of this span element. I don't think it needs a comment: I'm just doing something that all devs should do in this circumstance. Unless you think I should document this for junior HTML devs that haven't seen this before? In that case, let me know and I will add it in a future commit.